README.md

```diff
@@ -73,6 +73,8 @@ Create `${basedir}/.mvn/maven-git-versioning-extension.xml`.
 
 - *optional* `<updatePom>` global enable(`true`)/disable(`false`) version update in original pom file.
 
+- *optional* `<preferTags>` set `true` to prefer tag rules over branch rules if both match.
```

Comment:
may you can change it to 
- *optional* `<preferTags>` global enable(`true`)/disable(`false`) prefer tag rules over branch rules.`